.github/workflows/build.yml

```diff
@@ -61,15 +61,21 @@ jobs:
         wavpack
       VCPKG_DEFAULT_TRIPLET: ${{ matrix.vcpkg_triplet }}
       VCPKG_OVERLAY_PORTS: overlay/ports
+      DEPS_BASE_NAME: mixxx-deps
```

Comment:
We could go back to the even shorter `buildenv` name. :D

Reply:
I don't really mind. To put Mixxx in the name may remind drive by contributors what the file is for when finding it in the Downloads folder. 

Please propose the final name and let's merge.